.github/workflows/test.yaml

```diff
@@ -65,12 +65,13 @@ jobs:
             }
           }' > ./axelar-chains-config/info/local.json
 
-      # Create .env file with default hardhat private key that's prefunded
+      # Create .env file with default hardhat private key that's prefunded, default salt is keccak256('salt')
       - name: Prepare .env
         run: |
           echo 'PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80' >> .env
           echo 'ENV=local' >> .env
           echo 'CHAINS=test' >> .env
+          echo 'SALT=0xa05e334153147e75f3f416139b5109d1179cb56fef6a4ecb4c4cbc92a7c37b70' >> .env
```

Comment:
what's this for? Also, salt is usually a string and we hash in the script. If it's only needed by a specific script, then it should be passed in there and not applied globally to all cmds below

Reply:
will change to provide both `rawSalt` and `salt` options